Master-Game/World/Map.cs

```diff
@@ -1,20 +1,34 @@
-﻿using System.Drawing;
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
 
 namespace MasterGame.World
 {
     public class Map
     {
         private BaseTile [,] CurrentMap;
+        //private List<List<BaseTile>> MyCurrentMap;
 
         public void LoadMap()
         {
-            CurrentMap = new BaseTile [4,4]
+            MapData loadedMap = LoadJson();
+            int Rows = loadedMap.rows;
+            int Cols = loadedMap.cols;
+
+
+            CurrentMap = new BaseTile[4,4]
```

Comment:
Make this of dynamic size by calling it this way.

CurrentMap = new BaseTile[rows, cols];

Reply:
Right, that's what I thought. But when I did that, I am given an error that says "a constant value is expected". From what I can gather, it seems to be a limitation with arrays.

Reply:
Maybe we should use the List class instead?